tests/components/hassio/test_init.py

```diff
@@ -34,8 +51,8 @@ async def test_setup_api_ping(hass, aioclient_mock):
         result = await async_setup_component(hass, "hassio", {})
         assert result
 
-    assert aioclient_mock.call_count == 5
-    assert hass.components.hassio.get_homeassistant_version() == "10.0"
+    assert aioclient_mock.call_count == 6
+    assert hass.components.hassio.get_homeassistant_version() == "0.110.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)